chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/activity/GetActivityDetailsActionPermissions.java

```diff
@@ -244,12 +244,12 @@ private boolean hasAccessToAnnualReport(final Long reportId, final JsonNode annu
      * @param announcementJson
      * @return
      */
-    private boolean hasAccessToAnnouncement(final JsonNode announcementJson) {
+    private boolean hasAccessToAnnouncement(JsonNode announcementJson) {
         boolean hasAccess = false;
-        AnnouncementDTO announcement = null;
+        Announcement announcement = null;
         try {
-            announcement = jsonMapper.convertValue(announcementJson, AnnouncementDTO.class);
-        } catch (final Exception ex) {
+            announcement = jsonMapper.convertValue(announcementJson, Announcement.class);
+        } catch (Exception ex) {
             LOGGER.error("Could not parse announcement activity as AnnouncementDTO." + "JSON was: " + announcementJson,
```

Comment:
Could you just change the log message to not say "AnnouncementDTO" as well? I know it's a small thing